v2/options.go

```diff
@@ -48,7 +48,7 @@ type ReadWriteOption interface {
 // padding begins.
 func ZeroLengthSectionAsEOF(enable bool) ReadOption {
 	return func(o *ReadOptions) {
-		o.ZeroLengthSectionAsEOF = true
+		o.ZeroLengthSectionAsEOF = enable
```

Comment:
nice catch :)

Reply:
🤦

Reply:
To clarify this did not cause the issue; it just meant if the option is present it's always enabled. Totally my fault for missing it in the PR review.

Reply:
Yeah ofcourse, I agree this wasn't the RC.